First Commit

zenopopovici opened the pull request.

raresraulea committed:
First Commit

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,168 @@
+                            <form class="nav__item--form">
+                                <input class="nav__item--input" type="search">
+                                <button name="submit" class="nav__item--button">
+                                    <span class="fas fa-search"></span>
```
Teoretic trebuia icoana aici svg, dar dat fiind ca nu e design e ok

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,168 @@
+                   <blockquote class="article__content--blockq">
+                       <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque doloremque aperiam explicabo voluptatem dicta. Similique aut illo, at nulla totam iusto sapiente dicta, nisi expedita placeat optio. Id, recusandae nostrum. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum nam fugit dicta laboriosam nostrum explicabo sunt adipisci.</p>
+                       <cite class="blockquote__cite">
+                           <p>"</p>
```
Asta se adauga prin css pseudo elements

https://learn-the-web.algonquindesign.ca/topics/before-after/

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,168 @@
+                   <h2>Heading 2</h2>
+                   <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tenetur amet labore sapiente unde pariatur reiciendis provident doloremque assumenda beatae aspernatur omnis nulla in saepe porro similique ipsam inventore, neque cumque?. Lorem ipsum dolor sit amet consectetur adipisicing elit. <a href="#">Link example</a>. Quae, facere nobis blanditiis ullam eligendi amet recusandae ...
+                   <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum perspiciatis nam, aliquid magnam perferendis excepturi ad, doloribus dolorem iure minima dolor molestias. Nihil impedit, vel facilis repudiandae enim doloremque ex.. Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores quaerat tempore possimus nam! Dignissimos, in esse? Deserunt inventore ex totam v ...
+                   <blockquote class="article__content--blockq">
```
Nu se prescurteaza class name. Nu e folosit corect aici. Ar trebui sa fie `article__blockquote blockquote`

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,168 @@
+                        </cite>
+                   </blockquote>
+                   <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Et ad porro cupiditate sequi odit nam doloribus magnam magni eaque consectetur unde adipisci veniam praesentium, rerum aspernatur sit quis laborum ut! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio, similique tempora molestias suscipit modi impedit nemo voluptas. Magnam corrupti maiores quo delectus. Ra ...
+                   <img class="article__content--img" src="https://via.placeholder.com/725x375?text=Article+Image" alt="Placeholder Image" width="725" height="375">
```
`article__image`

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+body{
```
Intre selector si { trebuie spatiu.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
```
`font-family trebuie sa aiba fallback: https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background: #000;
```
Schimba doar culoarea `background-color`. Altfel risti sa suprascrii background-ul setat de altcineva.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+    display: flex;
+    flex-direction: column;
+}
+.header{
```
linie libera intre selectori

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,168 @@
+                   </li>
+                   <li class="nav__item">
+                        <a href="#" class="nav__link">
+                            <form class="nav__item--form">
```
Asta trebuie sa fie `nav__form` citeste inca o data BEM: https://en.bem.info/methodology/html/

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,168 @@
+                        <a href="#" class="nav__link">
+                            <form class="nav__item--form">
+                                <input class="nav__item--input" type="search">
+                                <button name="submit" class="nav__item--button">
```
- Buton tb sa aiba `type="submit"`, name-ul nu e relevant pe buton ca nu trebuie sa se trimita prin formular.

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,168 @@
+                   <li class="nav__item">
+                        <a href="#" class="nav__link">
+                            <form class="nav__item--form">
+                                <input class="nav__item--input" type="search">
```
Input-ul tb sa aiba name. clasa corecta e `nav__input`

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+}
+.nav__item--input{
+    border: none;
+    outline: none;
```
Nu scoate outline-ul de pe input-uri, daca il scoti nu vor mai fi navigabile vizibil din tastatura. Outline-ul tb inlocuit cu ceva.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+    flex-direction: column;
+    align-items: center;
+    width: 60%;
+    padding: 3% 0;
```
Padding-ul nu se da niciodata relativ.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+}
+.article__content--img{
+    width: 100%;
+    margin: 3% 0px;
```
`0` nu `0px`

zenopopovici replied:
marginea nu ar trebui data in valori relative.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+    position: relative;
+    background-color: #fff;
+    flex: 1;
+    transition: all 0.3s;
```
Pune tranzitie doar pe ce vrei sa animezi, nu pe toate, e foarte taxing pentru browser.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+    background-image: url("https://via.placeholder.com/100x150?text=Article");
+}
+.related__item:hover{
+    outline: 3px solid #2b76d3;
```
Asta se face cu `border` sau `box-shadow` nu cu outline.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,311 @@
+    .header__logo{
+        display: none;
+    }
+}
```
Vezi ca ceva nu merge bine aici cand redimensionezi.

raresraulea committed:
Update 1 index.html dupa observatii

raresraulea committed:
Update1 main.css dupa observatii

zenopopovici commented on main.css:
```diff
@@ -53,108 +60,137 @@ body{
     position: absolute;
-    top: 30px;
-    left: -100px;
+    top: 3em;
```
Why em? and not 48px :)

zenopopovici commented on main.css:
```diff
@@ -53,108 +60,137 @@ body{
+.article__img {
     width: 100%;
-    margin: 3% 0px;
+    margin: 2em 0;
```
why em and not 32px?

zenopopovici commented on main.css:
```diff
@@ -53,108 +60,137 @@ body{
+.article__blockquote::before {
+    content: open-quote;
+    position: absolute;
+    margin-left: -1.5em;
```
Why em?

zenopopovici commented on main.css:
```diff
@@ -214,52 +258,61 @@ body{
 }
-.nav-secondary__link{
+
+.nav-secondary__link {
```
Keep properties sorted (eg. move after `nav-secondary__wrapper`).

zenopopovici reviewed (commented):
Good improvements 👍

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+          width="160"
+          height="50"
+          class="footer__logo"
+        />
```
Nu e nevoie sa inchizi tag-ul de imagine, e self closing.
```suggestion
        >
```
```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+            alt="Placeholder Image"
+            width="725"
+            height="375"
+          />
```
Self closing
```suggestion
          >
```
```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
```
Self closing.
```suggestion
    >

```

raresraulea replied:
Nu stiu de unde au aparut closing tagurile astea. Eu nu le am pus. Am dat un beautify. Poate de acolo?

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+      content="Blogul pasionatilor de biciclete din Sibiu"
+    />
+    <title>VeloPlus</title>
+    <link href="main.css" rel="stylesheet" />
```
```suggestion
    <link href="main.css" rel="stylesheet">

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
```
```suggestion
    <meta charset="utf-8">

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="author" content="Zeno" />
```
```suggestion
    <meta name="author" content="Rares Raulea">

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+  <head>
+    <meta charset="utf-8" />
+    <meta name="author" content="Zeno" />
+    <meta name="keywords" content="velo magazin biciclete blog" />
```
```suggestion
    <meta name="keywords" content="velo magazin biciclete blog">

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+    <meta
+      name="description"
+      content="Blogul pasionatilor de biciclete din Sibiu"
+    />
```
```suggestion
    >

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+          alt="Logo VeloPlus"
+          width="170"
+          height="60"
+        />
```
```suggestion
        >

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+            Dignissimos, in esse? Deserunt inventore ex totam vero repudiandae.
+            Consectetur saepe ex, cupiditate earum animi reprehenderit dolores.
+          </p>
+          <blockquote class="article__blockquote">
```
```suggestion
          <blockquote class="article__blockquote blockquote">

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+            quisquam exercitationem ad similique.
+          </p>
+          <img
+            class="article__img"
```
```suggestion
            class="article__image"

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+              width="130"
+              height="35"
+              class="social__image"
+            />
```
```suggestion
            >

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+              width="105"
+              height="35"
+              class="social__image"
+            />
```
```suggestion
            >

```

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,215 @@
+      <div class="footer__nav nav-secondary">
+        <ul class="nav-secondary__wrapper">
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Biciclete </a>
```
```suggestion
            <a href="#" class="nav-secondary__link">Biciclete </a>

```

Daca ai nevoie de spatiu inainte foloseste margin / padding in css. Spatiul lasat inainte poate avea diferite valori in diferite browsere.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,369 @@
+            rgba(0, 0, 0, 0.65),
+            rgba(0, 0, 0, 0.95),
+            rgba(0, 0, 0, 1)),
+        url("bycicle-bg2.jpg");
```
```suggestion
        url('bycicle-bg2.jpg');

```

Hotaraste-te ce fel de ghilimele folosesti. Simple sau duble.

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,369 @@
+    position: relative;
+}
+
+/* .article */
```
```suggestion

```

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,369 @@
+    min-height: 320px;
+    padding: 10px 20px;
+    border: 3px solid transparent;
+    background-image: url("https://via.placeholder.com/100x150?text=Article");
```
```suggestion
    background-image: url('https://via.placeholder.com/100x150?text=Article');

```

zenopopovici commented on main.css:
```diff
@@ -0,0 +1,369 @@
+        margin: 20px 20px;
+        min-height: 320px;
+        padding: 10px 20px;
+        background-image: url("https://via.placeholder.com/100x150?text=Article");
```
```suggestion
        background-image: url('https://via.placeholder.com/100x150?text=Article');

```

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update main.css

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update main.css

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update main.css

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update main.css

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update index.html

Co-authored-by: Zeno Popovici <zeno@graffino.com>

raresraulea committed:
Update main.css

raresraulea committed:
Update main.css

raresraulea committed:
App.js 

Contine o functie pentru a face meniul 100% responsive

raresraulea committed:
Legare script

raresraulea committed:
Reformatare

raresraulea committed:
responsive update + corectat nume clasa

Testez commituri din aplicatia GitHub Desktop
-corectat nume clasa .article-image
-modificare interval responsive media query
-modificare latime .article-content pentru tablete si telefoane

raresraulea committed:
Add files via upload

raresraulea committed:
Fixed navi, higher hero image

raresraulea committed:
Merge branch 'develop' of https://github.com/raresraulea/VeloPlus into develop

raresraulea committed:
hover effect with pseudo-elements

nu reusesc sa fac opacitatea background-image-ului sa se modifice smooth...

raresraulea committed:
Animatii CSS

1. animatie hover sectiunea Related cu pseudo-elements
2. animatie hover meniu
3. animatie de intrare pentru elementele din header
4. deschidere imagine din articol ca in galerie + inchidere de pe X

raresraulea committed:
box-shadow nav + nav item hover

zenopopovici commented on app.js:
```diff
@@ -0,0 +1,44 @@
+const navigationSlideIn = () => {
+
+    console.log("CALL")
+    const navLinks = document.querySelector('.nav__wrapper');
```
Noi in general folosim selectori separati pentru JS care incep cu `.js-blabla` in felul asta daca se modifica cod-ul sau css-ul nu se strica functionalitatea JS.

zenopopovici commented on app.js:
```diff
@@ -0,0 +1,44 @@
+const navigationSlideIn = () => {
+
+    console.log("CALL")
```
Remove console.logs. Used just for debug.

zenopopovici commented on index.html:
```diff
@@ -0,0 +1,231 @@
+        <span class="footer__text">Copyright VeloPlus &copy; 2021</span>
+      </div>
+    </footer>
+    <script src="app.js"></script>
```
Tb sa ai atribut de async pe script https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script

raresraulea commented:
Salut, Zeno,

am citit despre atributele async și defer și despre cum se încarcă normal
vs async vs defer. Am înțeles următoarele:

1. Normal/Default

i.) Se parsează documentul HTML de sus în jos, de la stânga la dreapta
(firesc)
ii.) În momentul în care este întâlnit tagul de script, se oprește parsarea
iii.) Se începe descărcarea scriptului de JS
iv.) Imediat după ce e disponibil/gata descarcat, se executa scriptul
v.) După ce se termina execuția scriptului, se reia parsarea documentului
HTML

2. Async

i.) [...]
ii.) În momentul în care se întâlnește tagul de script cu atribut async, se
pornește în paralel descărcarea scriptului și se continua parsarea
iii.) În momentul în care se finalizează descărcarea scriptului, se oprește
parsarea si se executa scriptul
iv.) După finalizarea execuției scriptului, se reia parsarea documentului
HTML

3. Defer

i.) [...]
ii.) În momentul în care se întâlnește tagul de script cu atribut async, se
pornește în paralel descărcarea scriptului și se continua parsarea
iii.) În momentul în care se finalizează descărcarea scriptului, NU se
oprește parsarea, ci se continua parsarea documentului HTML
iv.) După finalizarea parsarii documentului HTML, se executa IN ORDINE,
scripturile descărcate asincron/în paralel, dar totuși înainte sa fie
triggerit eventul de DOMContentLoaded. Deci, chiar dacă parsarea HTML e
finalizată, dacă scriptul conține un eventListener pe 'DOMContentLoaded',
nu e triggerit instant, ci doar la următoarea rulare după executarea
tuturor scripturilor descărcate. Dar oare ar trece de un
if(document.readyState != 'loading') sau e acel state de 'interactive'?

Mersi mult pentru implicare și ajutor!

Cu stima,

Rares Raulea


lun., 5 apr. 2021, 00:01 Zeno Popovici ***@***.***> a scris:

> Merged #1 <https://github.com/raresraulea/VeloPlus/pull/1> into main.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/raresraulea/VeloPlus/pull/1#event-4549507819>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGKS7WGUX74K2G5CQGGEAZ3THDHUJANCNFSM42AF7CMQ>
> .
>
